org.opent2t.sample.hub.superpopular/org.opent2t.sample.hub.superpopular.json

```diff
@@ -0,0 +1,24 @@
+{
+  "id": "http://schemas.opentranslatorstothings.org/org.opent2t.sample.hub.superpopular#",
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "description": "",
+  "title": "OpenT2T Hub",
+  "definitions": {
+    "org.opent2t.sample.hub.superpopular": {
+      "type": "object",
+      "properties": {
+        "name": {
+          "description": "ReadOnly, Gets the name of the hub."
+        },
+        "devices": {
+          "description": "ReadOnly, Gets the current set of devices associated with the hub."
+        }
```

Comment:
'devices' still needs a type.

---
In reply to: [76154047](https://github.com/openT2T/translators/pull/75#discussion_r76154047) [](ancestors = 76154047)